lib/avail.rb

```diff
@@ -0,0 +1,63 @@
+# frozen_string_literal: true
+
+require 'json'
+require 'net/http'
+
+module Avail
+  PVTA_BASE_API_URL = 'https://bustracker.pvta.com/InfoPoint/rest'
+
+  # Returns a hash keyed by stop.
+  # Each value is a hash mapping from route direction data to the next time.
+  # The route direction data is route and headsign.
+  def self.next_departures_from(stops, after:)
+    routes = Hash[Route.all.map { |r| [r.avail_id, r] }]
+    times = {}
+    stops.each do |stop|
+      uri = departures_uri(stop.hastus_id)
+      stop_departure = JSON.parse(Net::HTTP.get(uri)).first
+      route_directions = stop_departure.fetch 'RouteDirections'
+      route_directions.each do |route_dir|
```

Comment:
I don't know if I would call this `route_dir`. With context, it makes sense that you mean route direction, but if I were to glance at this, I might think you mean a route directory.

Reply:
That's not unreasonable, but on the other hand, it's `some_plural_noun.each`. Typically I think you'd expect it to be an instance of whatever the plural noun is.
Do you have a suggestion for something clearer I could use?

Reply:
Nah. It's fine as is if you think it's fine.